conf/test.config

```
@@ -42,6 +41,9 @@ params {
     genome_notes_api            = "https://notes-staging.tol.sanger.ac.uk/api/v1"
     note_template               = "${projectDir}/assets/genome_note_template.docx"
 
+    // Contact Map Options
+    select_contact_map          = "pretext"
```

I'd want to test both for now
```suggestion
    select_contact_map          = "both"
```

---

Done